README.md

```diff
@@ -65,7 +68,7 @@ This README is just the tip of the iceberg! **Check out our full documentation s
 
 There you will find a conceptual system walkthrough, tips on writing test plans, instructions on running test plans, configuring runners and builders, deploying Kubernetes clusters, and a lot more.
 
-_NOTE: the documentation is work-in-progress (is it ever done?). Please report any problems or inaccuracies by [opening a docs issue on this repo](https://github.com/testground/testground/issues/new?assignees=&labels=docs&template=DOCS.md&title=docs%20site:%20%3Cdescribe%20the%20problem%3E)._
+Please report any problems or inaccuracies by [opening a docs issue on this repo](https://github.com/testground/testground/issues/new?assignees=&labels=docs&template=DOCS.md&title=docs%20site:%20%3Cdescribe%20the%20problem%3E).
```

Comment:
It is nice that we explain to folks how to improve the docs, but it is implicit that the documentation is work-in-progress. The codebase is also work-in-progress, but we don't specify that.